torch_xla/core/xla_model.py

```diff
@@ -990,14 +1042,13 @@ def reduce_gradients(optimizer, groups=None, pin_layout=True):
   """
   count = xrt_world_size()
   if count > 1:
-    gradients = _fetch_gradients(optimizer)
```

Comment:
Can we keep the original behavior? And maybe use a flag to turn this feature on?

Reply:
OK..let me work on that

Reply:
Maybe we should introduce an argument "bucket_cap_mb" that turns this on, instead of environmental variable?  bucket_cap_mb=0 turns off bucketing and is the default?

Reply:
done